dotcom-rendering/src/experiments/tests/ad-block-ask.ts

```diff
@@ -5,7 +5,7 @@ export const adBlockAsk: ABTest = {
 	author: '@guardian/commercial-dev',
 	start: '2024-04-10',
 	expiry: '2024-05-31',
-	audience: 0 / 100,
+	audience: 10 / 100,
 	audienceOffset: 0 / 100,
```

Comment:
Assuming you’ve checked [this is the correct offset](https://frontend.gutools.co.uk/analytics/abtests) for this cohort.